views/advanced_usage.dt

```diff
@@ -34,7 +34,7 @@ block body
 
 	dl
 		dt <code>dub add-path</code>
-		dd Adds a local search directory. All direct subdirectories will be searched for packages. Local search directories are preferred over packages downloaded from the registry.
+		dd Adds a local search directory. All direct subdirectories will be searched for packages. Local search directories are manual overrides over the packages downloaded from the registry, so they will always be searched before looking into the registry.
```

Comment:
I think just "Local search directories are always searched before looking up packages from the registry", although to be honest I'm not sure this paragraph needs changing.

Reply:
"Preferred" could mean that dub people are recommending to use local packages instead of those on registry, but I can change for that always searched thing